package.json

```diff
@@ -43,7 +43,8 @@
     "webgl-distort": "jywarren/webgl-distort#*"
   },
   "engines": {
-    "yarn": ">= 1.0.0"
+    "yarn": ">= 1.0.0",
+    "node": ">=10.0.0"
```

Comment:
Let's mention this in readme?

Reply:
@sagarpreet-chadha this was actually a non essential change ie. can be removed and won't change anything. Serves more as a hint I think. What fixed the problem was telling yarn to ignore engines in `.travis.yml`